### PR TITLE
Added external secret for prometheus

### PR DESCRIPTION
**Pull request by bamachrn**
This PR adds external secrets from vault to prometheus operator

### CODE REVIEW

File: components/monitoring/prometheus/stone-stg-m01/monitoring-prometheus-oauth-config-path.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+- op: add
```
Comment (gbenhaim) [inline components/monitoring/prometheus/stone-stg-m01/monitoring-prometheus-oauth-config-path.yaml:2]: folder for specific cluster should go under a folder for an environment, for example
`components/gitops/staging/stone-stg-m01/`

**Review by gbenhaim — CHANGES_REQUESTED** [review]

File: components/monitoring/prometheus/base/external-secrets/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```
Comment (gbenhaim) [inline components/monitoring/prometheus/base/external-secrets/kustomization.yaml:2]: in development, the external-secrets-opeator isn't deployed (at least for now), so I suggest not putting crds it consumes in base.

**Comment by bamachrn** [comment]
@gbenhaim could you please have a look on this now?

**Review by gbenhaim — APPROVED** [review]

**Pull request merged**